notification2-examples/hello-world-microservice/pom.xml

```diff
@@ -3,21 +3,20 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>c8y.example.notification</groupId>
+        <artifactId>notification2-examples</artifactId>
+        <version>${revision}${changelist}</version>
+    </parent>
 
-    <groupId>c8y.example</groupId>
-    <artifactId>hello-world-notification-microservice</artifactId>
-    <name>Cumulocity :: Examples :: Hello World Notification 2.0 Microservice</name>
-    <version>${revision}${changelist}</version>
+    <artifactId>hello-world-microservice</artifactId>
```

Comment:
rename to `hello-notification`, then you don't have to change the docker image name in the `microservice-package-maven-plugin`

Reply:
But no issues if it is left as it is right as image name is explicitly provided?

Reply:
How about I set it to hello-world-microservice?

Reply:
Done.